flang/include/flang/Frontend/FrontendAction.h

```diff
@@ -112,6 +112,10 @@ class FrontendAction {
   // Run semantic checks for the current input file. Return False if fatal
   // errors are reported, True otherwise.
   bool RunSemanticChecks();
+  // Generate run-time type information for derived types. This may lead to new
+  // semantic errors. Return False if fatal errors are reported, True
+  // otherwise.
```

Comment:
Why aren't you using doxygen in the driver to help document generation?

Reply:
These methods are `protected`, so I just used regular comments. I don't mind either way.